services/.env.example

```diff
@@ -153,3 +153,6 @@ MOCK_COMMERCIAL_BANK_ETHIOPIA=
 
 # Boolean to sync data automatically with third parties (now only used for Intersolve Visa)
 SYNC_WITH_THIRD_PARTIES=
+
+REDIS_HOST=
+REDIS_PORT=
```

Comment:
Can this be added somewhere higher up? (Close to other relevant ENV-variables)
+ Add some explanation in comments above it?